xcp-inventory.opam

```diff
@@ -7,18 +8,12 @@ dev-repo: "git://github.com/xapi-project/xcp-inventory.git"
 tags: [ "org:xapi-project" ]
 build: [
   ["./configure" "--default_inventory=%{prefix}%/etc"]
-  [make]
-]
-install: [
-  [make "install"]
-]
-remove: [
-  [make "uninstall"]
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
 ]
+build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
 depends: [
   "ocamlfind" {build}
-  "oasis" {build}
-  "ocamlbuild" {build}
+  "jbuilder" {build}
   "base-threads"
   "xapi-stdext"
```

Comment:
One last change. Can you please use the specific `Xapi_stdext_...` modules and import only the required one in the `jbuild` and `opam` file? As in https://github.com/xapi-project/xcp-networkd/pull/120